Description: To verify SSL certificates for HTTPS requests, use the bundle
 provided by ca-certificates instead of python-certifi. Since requests 0.11.2
 it is not needed to hard-code bundle path because requests will search for it.
Author: Daniele Tricoli <eriol@mornie.org>
Forwarded: not-needed
Last-Update: 2012-04-23

--- a/setup.py
+++ b/setup.py
@@ -21,7 +21,7 @@
     os.system('python tests/test_requests.py')
     sys.exit()

-required = ['certifi>=0.0.7',]
+required = []
 packages = [
     'requests',
     'requests.packages',
